source/extensions/resource_monitors/cgroup_memory/cgroup_memory_monitor.cc

```diff
@@ -0,0 +1,54 @@
+#include "source/extensions/resource_monitors/cgroup_memory/cgroup_memory_monitor.h"
+
+#include "envoy/common/exception.h"
+
+#include "source/common/common/assert.h"
+#include "source/common/common/thread.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ResourceMonitors {
+namespace CgroupMemory {
+
+CgroupMemoryMonitor::CgroupMemoryMonitor(
+    const envoy::extensions::resource_monitors::cgroup_memory::v3::CgroupMemoryConfig& config,
+    std::unique_ptr<CgroupMemoryStatsReader> stats_reader)
+    : max_memory_bytes_(config.has_max_memory_bytes() ? config.max_memory_bytes().value() : 0),
```

Comment:
prefer using  `PROTOBUF_GET_WRAPPED_OR_DEFAULT` here for getting the value of max_memory_bytes